frontend/src/services/roleBindingService.ts

```diff
@@ -1,16 +1,14 @@
-import axios from 'axios';
+import { k8sCreateResource, k8sGetResource } from '@openshift/dynamic-plugin-sdk-utils';
 import { RoleBinding } from '../types';
+import { RoleBindingModel } from '../models';
 
 export const getRoleBinding = (projectName: string, rbName: string): Promise<RoleBinding> => {
-  const url = `/api/rolebindings/${projectName}/${rbName}`;
-  return axios.get(url).then((response) => {
-    return response.data;
+  return k8sGetResource({
+    model: RoleBindingModel,
+    queryOptions: { name: rbName, ns: projectName },
   });
 };
 
 export const createRoleBinding = (data: RoleBinding): Promise<RoleBinding> => {
-  const url = `/api/rolebindings`;
-  return axios.post(url, data).then((response) => {
-    return response.data;
-  });
+  return k8sCreateResource({ model: RoleBindingModel, resource: data });
```

Comment:
The API calls change the way we get data -- with the SDK vs what we did in our custom API. Needed to convert this to help with the utility of verify resource.